test/k8s-integration/main.go

```diff
@@ -668,7 +675,20 @@ func runTestsWithConfig(testParams *testParameters, testConfigArg, reportPrefix
 			kubetestDumpDir = artifactsDir
 		}
 	}
-	ginkgoArgs := fmt.Sprintf("--ginkgo.focus=%s --ginkgo.skip=%s", testParams.testFocus, testParams.testSkip)
+
+	focus := testParams.testFocus
+	skip := testParams.testSkip
+	if testParams.snapshotClassFile == "" && !strings.Contains(skip, "VolumeSnapshotDataSource") {
```

Comment:
I think if the `snapshotClassFile` is not set, the k/k integration framework would not run snapshot tests? is that not the case